provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/Query.java

```diff
@@ -101,8 +100,7 @@
      * @param pageSize the {@code int} with the maximum number of items per iteration. It cannot be negative.
      * @throws IllegalArgumentException if one of the parameters is invalid.
      */
-    @SuppressWarnings("deprecation")
-    protected Query(ContractApiHttp contractApiHttp, String targetPath, QuerySpecification querySpecification, int pageSize)
+    Query(ContractApiHttp contractApiHttp, String targetPath, QuerySpecification querySpecification, int pageSize)
```

Comment:
what about the "deprecation"?

Reply:
This was a warning that I missed removing back when I was deleting deprecated methods. This method called into a deprecated method at one point, but no longer does, so we can safely remove this suppression